src/utils/workOrderActions.js

```diff
@@ -74,7 +74,7 @@ export const WORK_ORDER_ACTIONS = [
   {
     href: 'edit',
     title: 'Edit',
-    description: 'Edit Work Order description',
+    description: 'Edit Work Order',
```

Comment:
Currently, you can only edit the description. Isnt the current description more clear to the users?